spec/requests/membership_submissions_request_spec.rb

```diff
@@ -0,0 +1,55 @@
+require "rails_helper"
```

Comment:
I think this file is redundant with the next one? Only change is in the file name "request" ?

Reply:
Ah, you're right. This file should be in the commits. Let me remove it.

Reply:
_Update_: The redundant file has been removed.